scripts/apps/packaging/views/sd-package-item-preview.html

```diff
@@ -19,10 +19,12 @@
 	  <div ng-show=":: data.anpa_take_key">{{ :: data.anpa_take_key}}</div>
       <div ng-hide=":: data.headline || data.slugline || data.description_text" translate>Blank headline received</div>
     </div>
-
-    <div class="package-item__item-creator">
-      Created <time sd-datetime data-date="data.firstcreated" data-from-now="true"></time>
-   <span ng-if="userLookup[data.original_creator]" translate>by <b>{{userLookup[data.original_creator].display_name}}</b></span>
+    <div>
+      <span ng-if="label" class="state-label state-in_progress">label: <span id="package-item-label"><b>{{label.name}}</b></span></span>
```

Comment:
there is missing `translate` and there should be no id in `<span id="package-item-label"` because there can be multiple items in package with a label